README.md

```diff
@@ -0,0 +1,174 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse! <a name="about-project"></a>
+
+**Hello Microverse** is a project to configure linters, HTML & CSS files.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+*Web browser like  <a href='https://www.google.com/chrome/'>Google Chrome</a>
+
+*<a href='https://git-scm.com/'>Git</a>
+
+*<a href='https://github.com/'>GitHub</a>
+
+*<a href='https://code.visualstudio.com/'>Source Code Editor</a>
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/JhonatanErosa/Project-1_Hello-Microverse
+
+
+### Install
+
+Install this dependencies:
+
+npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+*Clone the repo to your pc
+
+*Use the extension <a href='https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer'>Live Server
+</a> or open de .html file 
+
+### Run tests
+
+To run tests, run the following command:
+
+*HTML linters:
+  
+npx hint .
+
+*CSS linters:
+
+npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+You can deploy this project using:
+
+<a href='https://github.com/'>GitHub</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Jhonatan Ali Caamal Erosa**
+
+- GitHub: [@JhonatanErosa](https://github.com/JhonatanErosa)
+- Twitter: [@erosa_c](https://twitter.com/erosa_c)
+- LinkedIn: [Jhonatan Ali Caamal Erosa](https://www.linkedin.com/in/jhonatan-ali-caamal-erosa-bb3829271/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project support us with a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
```

Comment:
Lines like these are only provided in the Microverse Readme Template to guide you on how to modify the template to suit your project and so after you modify your Readme file, I think you can remove them since they don't give any information peculiar to your project.